Cargo.toml

```diff
@@ -24,25 +27,49 @@ repository = "https://github.com/graphprotocol/graph-node"
 license = "MIT OR Apache-2.0"
 
 [workspace.dependencies]
-diesel = { version = "2.1.3", features = ["postgres", "serde_json", "numeric", "r2d2", "chrono"] }
+anyhow = "1.0"
+async-graphql = { version = "7.0.6", features = ["chrono", "uuid"] }
+async-graphql-axum = "7.0.6"
+axum = "0.7.5"
+chrono = "0.4.38"
+clap = { version = "4.5.4", features = ["derive", "env"] }
+constant_time_eq = "0.3.0"
```

Comment:
Do we really need this? Where it's used, to compare tokens, doesn't seem all that time critical. And looking at the source, I don't see how it compares strings in constant time

Reply:
Fixed by the refactor